maven-embedder/src/test/java/org/apache/maven/cli/MavenCliTest.java

```diff
@@ -197,6 +197,7 @@ public void testMavenConfigInvalid()
      * <pre>
      *   -T 3
      *   -Drevision=1.3.0
+     *   -Dlabel=Apache Maven
```

Comment:
I propose to use `-Dlabel="Apache Maven"` or `"-Dlabel=Apache Maven"`, as you would never be able to use `-Dlabel=Apache Maven` from commandline.

Reply:
Fully agree, will change